Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.0)
+    rspec-mocks (3.9.1)
```

Comment:
### [rspec-mocks](https://github.com/rspec/rspec-mocks)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.9.0 → 3.9.1

[_[change-log](https://github.com/rspec/rspec-mocks/blob/v3.9.1/Changelog.md), [source-code](https://github.com/rspec/rspec-mocks)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-mocks/compare/v3.9.0...v3.9.1).

These are the individual commits:
- (24ec2d4) [Updates from rspec-dev (2019-12-26) (#1305)](https://github.com/rspec/rspec-mocks/commit/24ec2d487afdda277c26069c7db2f09d483b48bc)
- (dc92458) [#1308: Call when_declaring_verifying_double for any_instance…](https://github.com/rspec/rspec-mocks/commit/dc924582320284ce58ede743b2bc29150a00223a)
- (201355a) [Changelog for #1309](https://github.com/rspec/rspec-mocks/commit/201355aeb54d66126cd4422285c142eb2c6e1cd9)
- (f45810a) [Version 3.9.1](https://github.com/rspec/rspec-mocks/commit/f45810aa3add135bd8235df600bbe4b446fc77d4)


</details>